README.md

```diff
@@ -1,307 +1,29 @@
-# Elastic Enterprise Search connectors
+# Elastic connectors
 
-![logo](logo-enterprise-search.png)
```

Comment:
Why lose the logo? I love having the branding.

Reply:
Although it still requires Enterprise Search to run connectors, and it's still documented under the [Enterprise Search guide](https://www.elastic.co/guide/en/enterprise-search/current/connectors.html) (But it's named as `Elastic connectors`), I'm not sure we still want to make it under Enterprise Search. After all, we want to move Enterprise Search out of the critical execution path.

Reply:
Ah! Ok this might be a question then for @danajuratoni and @serenachou. I think we still want to market these as being part of Enterprise Search. Elastic Enterprise Search is now supposed to represent pretty much any Elastic business that's _not_ Security or Observability. So it extends well beyond the borders of the Enterprise Search Rails Server. So I don't necessarily think it's _wrong_ to use the Enterprise Search branding here. But I also don't think it would be wrong to switch it with Elastic branding. WDYT?

Reply:
>I think we still want to market these as being part of Enterprise Search.

I think you are right. We can remove it when it's removed from the Enterprise Search guide.

I will add it back.